test/e2e/nvidia-gpus.go

```diff
@@ -158,10 +159,16 @@ func testNvidiaGPUsOnCOS(f *framework.Framework) {
 }
 
 // dsFromManifest reads a .json/yaml file and returns the daemonset in it.
-func dsFromManifest(fileName string) *extensions.DaemonSet {
+func dsFromManifest(url string) *extensions.DaemonSet {
 	var controller extensions.DaemonSet
-	framework.Logf("Parsing ds from %v", fileName)
-	data := generated.ReadOrDie(fileName)
+	framework.Logf("Parsing ds from %v", url)
+
+	response, err := http.Get(url)
```

Comment:
Should we retry a few times just to avoid spurious network issues or github serving issues?